src/sections/index.ts

```diff
@@ -9,3 +9,6 @@ export * from './TrustedByOpenSourceSection';
 export * from './PricingSection';
 export * from './FAQSection';
 export * from './Footer';
+export * from './FormRelatedIssues';
+export * from './PowerfulFeatures';
+export * from './WhatDoYouGet';
```

Comment:
can you create one section which will receive cards list, id, heading and sometimes description as a props? for me this section should work like `<CTASection />`